pkgs/applications/virtualization/docker/compose_1.nix

```diff
@@ -13,7 +13,7 @@ buildPythonApplication rec {
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "sha256-TIzZ0h0jdBJ5PRi9MxEASe6a+Nqz/iwhO70HM5WbCbc=";
+    hash = "sha256-TIzZ0h0jdBJ5PRi9MxEASe6a+Nqz/iwhO70HM5WbCbc=";
```

Comment:
This change doesn't seem related to awsebcli.

Reply:
It is related, because awsebcli created a function to update the hash and I converted it to overwrite hashes. It is also used to overwrite the hash for docker-compose.